clumper/clump.py

```diff
@@ -62,6 +63,56 @@ def read_json(cls, path):
             return Clumper(data)
         return Clumper(json.loads(pathlib.Path(path).read_text()))
 
+    @classmethod
+    def read_csv(cls, path, delimiter=",", fieldnames=None, nrows=None):
+        """
+        Reads in a csv file. Can also read files from url.
+
+        Parameters
```

Comment:
The rest of the docs refer to parameters as "arguments" and also have a slightly different markdown syntax. This is required by the mkdocs plugin that we're using here.